api/anilist/src/main/kotlin/com/imashnake/animite/api/anilist/sanitize/profile/Viewer.kt

```diff
@@ -0,0 +1,47 @@
+package com.imashnake.animite.api.anilist.sanitize.profile
+
+import com.imashnake.animite.api.anilist.ViewerQuery
+
+/** Sanitized [Viewer] */
```

Comment:
Shouldn't this be describing what a Viewer is?

Reply:
Apollo already generated Kdocs for most models/properties, I'm just redirecting 😛